eng/build.yml

```diff
@@ -168,3 +168,12 @@ phases:
         publishLocation: Container
       continueOnError: true
       condition: always()
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/7.0.1xx'))) }}:
```

Comment:
You may want to move this to a scheduled pipeline to avoid overwhelming the loc team. At their recommendation, we created a separate https://github.com/dotnet/aspnetcore/blob/main/.azure/pipelines/localization.yml file and associated pipeline. But dotnet/runtime does something similar to what you're doing here (see https://github.com/dotnet/runtime/blob/66af85735dc40dc7aa257329c91e790b7418b429/eng/pipelines/runtime-official.yml#L48).

/cc @jonfortescue @cristianosuzuki77

Reply:
Does it overwhelm the loc team if there aren't very many string changes (even if there are a lot of builds
)? Thinking on this, the installer repo itself actually has fairly few PRs. SDK does this on every build already and so perhaps a scheduled pipeline makes more sense for that repo than here.